client/app/helpers/helpers.md

```diff
@@ -0,0 +1,52 @@
+## Helpers
```

Comment:
Love the inclusion of this readme file but as a first-time viewer I'm having trouble understanding it intuitively. Would you mind adding more explanation?

Reply:
Yes, Can do!

Reply:
I added some intro text. Not sure if that helps, let me know what you think can make it clearer!

Reply:
Thank you for your thoughtfulness of developer experience in including this